include/llbuild/BuildSystem/BuildDescription.h

```diff
@@ -149,6 +149,34 @@ class Node {
       ArrayRef<std::pair<StringRef, StringRef>> values) = 0;
 };
 
+/// Definition for a file system stat used by the build file.
+class Stat {
+  // DO NOT COPY
+  Stat(const Stat&) LLBUILD_DELETED_FUNCTION;
```

Comment:
We really should just have a macro for `delete`ing these functions, like `LLBUILD_DISABLE_COPY(typename)`

Reply:
(though out of scope for this change, indeed this specific code will be removed in a moment)